pmoves/tools/hf_model_onboard.py

```
@@ -289,7 +312,46 @@ def upsert_to_supabase(fragment: AgentCardFragment) -> bool:
+        resp = httpx.post(
+            f"{supabase_url}/rest/v1/model_candidates?on_conflict=hf_id,intended_lane",
+            headers=headers,
+            json=candidate,
+            timeout=30,
+        )
+        resp.raise_for_status()
+        return True
+    except ImportError:
+        print("httpx not installed. Install with: uv pip install httpx", file=sys.stderr)
+        return False
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)